public/app/percona/pmm-dump/PMMDump.tsx

```diff
@@ -0,0 +1,328 @@
+import { HorizontalGroup, Icon, useStyles2, Badge, BadgeColor, LinkButton, Button } from '@grafana/ui';
+import appEvents from 'app/core/app_events';
+import { Page } from 'app/core/components/Page/Page';
+import { DATA_INTERVAL } from 'app/percona/backup/components/BackupInventory/BackupInventory.constants';
+import { DetailedDate } from 'app/percona/backup/components/DetailedDate';
+import { useRecurringCall } from 'app/percona/backup/hooks/recurringCall.hook';
+import { Action } from 'app/percona/dbaas/components/MultipleActions';
+import { DumpStatus, DumpStatusColor, DumpStatusText, PMMDumpServices } from 'app/percona/pmm-dump/PmmDump.types';
+import { DetailsRow } from 'app/percona/shared/components/Elements/DetailsRow/DetailsRow';
+import { ExtendedColumn, FilterFieldTypes, Table } from 'app/percona/shared/components/Elements/Table';
+import { deletePmmDumpAction, fetchPmmDumpAction } from 'app/percona/shared/core/reducers/pmmDump/pmmDump';
+import { getDumps } from 'app/percona/shared/core/selectors';
+import { isApiCancelError } from 'app/percona/shared/helpers/api';
+import { getExpandAndActionsCol } from 'app/percona/shared/helpers/getExpandAndActionsCol';
+import { logger } from 'app/percona/shared/helpers/logger';
+import { dateDifferenceInWords } from 'app/percona/shared/helpers/utils/timeRange';
+import { useAppDispatch } from 'app/store/store';
+import { useSelector } from 'app/types';
+import { ShowConfirmModalEvent } from 'app/types/events';
+
+import { Messages } from './PMMDump.messages';
+import { PMMDumpService } from './PMMDump.service';
+import { getStyles } from './PmmDump.styles';
+import { SendToSupportModal } from './SendToSupportModal';
+import { PmmDumpLogsModal } from './components/PmmDumpLogsModal/PmmDumpLogsModal';
+export const NEW_BACKUP_URL = '/pmm-dump/new';
+
+const pageNav: NavModelItem = {
+  icon: 'brain',
+  id: 'user-new',
+  text: 'PMM Dump',
+  subTitle:
+    'Simplify troubleshooting and accelerate issue resolution by securely sharing relevant data, ensuring a smoother support experience.',
+};
+
+export const PMMDump = () => {
+  const styles = useStyles2(getStyles);
+  const dispatch = useAppDispatch();
+  const { dumps } = useSelector(getDumps);
+  const [triggerTimeout] = useRecurringCall();
+  const [selected, setSelectedRows] = useState<Array<Row<PMMDumpServices>>>([]);
+  const [selectedDumpIds, setSelectedDumpIds] = useState<string[]>([]);
+  const [selectedDump, setSelectedDump] = useState<PMMDumpServices | null>(null);
+  const [isSendToSupportModalOpened, setIsSendToSupportModalOpened] = useState(false);
+  const [logsModalVisible, setLogsModalVisible] = useState(false);
+
+  const loadData = useCallback(async () => {
+    try {
+      await dispatch(fetchPmmDumpAction());
+    } catch (e) {
+      if (isApiCancelError(e)) {
+        return;
+      }
+      logger.error(e);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
I think it's fine having `dispatch` in dependency array.

Reply:
do we want to trigger loadData every time we dispatch something ?